containers/test/run.sh

```diff
@@ -0,0 +1,35 @@
+CONTENT=$HOME
+ENTRYPOINT=""
+if [ "$1" != "" ]; then
+  if [ "$1" != "shell" ]; then
+    CONTENT=$1
+    shift
+  fi
+  if [ "$1" == "shell" ]; then
+    ENTRYPOINT="/bin/bash"
+  fi
+fi
+
+# disable EULA for the sake of tests
+mkdir -p $CONTENT/datalab/.config/eula
+
+# On linux docker runs directly on host machine, so bind to 127.0.0.1 only
```

Comment:
@parthea Sorry that I was ambiguous.

Yes, binding to 127.0.0.1 is still required. Docker now supports that on all platforms it supports, so the check for linux is now obsolete and we can hardcode the portmap to "127.0.0.1:8081:8080"